### ISSUE TITLE
Offset in provided mapping files?

### ISSUE THREAD

**Issue by holynski** [IntelVCL/TanksAndTemples#16]
From the 'Download' page:

> The image sets are sampled at a frame rate of 1 fps from the video while the video was recorded with 29.97 fps. To find the corresponding frame F to the image I you need to calculate: **F = int(I*29.97)**, starting with I=0.

But, when looking into the 'training_data.zip' package, often times the values don't agree with this formula. For instance, in the 'Courthouse_mapping_reference.txt' file:
`...`
`19 568` <-- should be int(19 * 29.97) = 569
`20 598` <-- should be int(20 * 29.97) = 599
`21 628` <-- should be int(21 * 29.97) = 629
`...`

Is this mapping intentional? Am I missing something here?
I'm trying to test my own functions for generating a mapping file, but am having trouble recreating the mapping files that have been provided.

**Comment by arknapit** [comment]
Yes, you are right, I checked and it looks like there is an offset by 1 in some of the provided mapping files. I need to check if there is a reason for that or if it is by mistake. However, this mapping is just used for a rough alignment and our registration method is quite robust against small initial misalignments like that. The evaluation algorithm converges to the same F1 score even when you are several frames off. So if you try it out it should give you the same result with our mapping, and when using the mapping you proposed.
Let us know if it works.

**Comment by arknapit** [comment]
@holynski can this be closed?
